### PR TITLE
TH-782 | Accessibility/about page content from CMS

### PR DESCRIPTION
**Pull request by jorilindell**
## Description
Get about and accessibility pages from CMS
Remove static about pages

## Issues
[TH-782](https://helsinkisolutionoffice.atlassian.net/browse/TH-782)

### CODE REVIEW

File: src/domain/about/__tests__/AboutPage.test.tsx [hunk 1]
```diff
@@ -1,27 +1,73 @@
...
   act(() => {
     i18n.changeLanguage('fi');
   });
 
-  expect(screen.getAllByText('Tietoa palvelusta')).toHaveLength(1);
+  expect(
+    screen.getAllByText(
+      fakeAboutPagesResponse.data[0].headingSection?.fi as string
+    )
+  ).toHaveLength(1);
```
Comment (Sartonon) [inline src/domain/about/__tests__/AboutPage.test.tsx:37]: is there a reason to not use screen.getByText()?

`expect(
    screen.getByText(
      fakeAboutPagesResponse.data[0].headingSection?.fi as string
    )
  ).toBeInTheDocument()`

Comment (Sartonon) [reply src/domain/about/__tests__/AboutPage.test.tsx:37]: if not, I would change this to make it simpler

File: src/domain/accessibility/AccessbilityPage.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      className={styles.accessibilityPage}
+      title="accessibility.title"
+    >
+      <MainContent offset={-70}>
+        <LoadingSpinner isLoading={loading}>
+          {headingSection && (
+            <Hero>
+              <div
+                dangerouslySetInnerHTML={{
+                  __html: sanitizeHtml(headingSection),
```
Comment (Sartonon) [inline src/domain/accessibility/AccessbilityPage.tsx:36]: Hmm, is this sanitization needed here? The data come from trusted server? Or are there some other security concerns?

Maybe it is better to be safe than sorry :) but the downside is performance / sanitizeHtml package size

File: src/domain/about/__tests__/AboutPage.test.tsx (hunk 1 above)
Comment (jorilindell) [reply src/domain/about/__tests__/AboutPage.test.tsx]: Simplified tests

File: src/domain/accessibility/AccessbilityPage.tsx (hunk 2 above)
Comment (jorilindell) [reply src/domain/accessibility/AccessbilityPage.tsx:36]: In this particular case the data is coming from our CMS. Same sanitize method is also used on Events page and I don't trust LinkedEvents data importers or actually source data so much. (So we need the package anyway) The data amount is not so huge so I don't see that performance is big problem here.

Comment (Sartonon) [reply src/domain/accessibility/AccessbilityPage.tsx:36]: Ok sounds reasonable :)

**Review by Sartonon — APPROVED** [review]

**Pull request merged**